build-tools/src/testFixtures/groovy/org/elasticsearch/gradle/fixtures/AbstractGradleFuncTest.groovy

```diff
@@ -151,4 +162,29 @@ abstract class AbstractGradleFuncTest extends Specification {
             System.err.println("Syserr: " + proc.errorStream.text)
         }
     }
+
+    def cleanup() {
+        if (Boolean.getBoolean('test.keep.samplebuild')) {
```

Comment:
TIL this existed.

Reply:
Sorry, I wasn't clear - I didn't know that `Boolean.getBoolean(String)` existed 😅